google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -126,6 +126,8 @@ appengine.flex.deployment.cost.warning.afterLink=for pricing information.
 appengine.flex.deployment.appyaml.edit.button.text=edit
 appengine.flex.deployment.docker.directory.label=Docker directory:
 appengine.flex.deployment.use.different.docker.directory=Use a different Docker directory
+appengine.flex.deployment.staged.artifact.name.label=Staged artifact name:
+appengine.flex.deployment.staged.artifact.name.tooltip=The filename to use when staging your artifact for deployment. Directories are allowed; all paths are relative to the temporary staging directory.
```

Comment:
hmm, how about something like

> The filename to use when staging your artifact for deployment in the staging directory. For subdirectories, include path relative to the staging directory.

Still not perfect, but maybe an improvement.